Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
```

Comment:
### [diff-lcs](https://github.com/halostatue/diff-lcs)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.3 → 1.5.0

[_~~change-log~~, [source-code](https://github.com/halostatue/diff-lcs), [gem-diff](https://my.diffend.io/gems/diff-lcs/1.1.3/1.5.0)_]





<details>
<summary>Commits</summary>

A change of **143** commits. See the full changes on [the compare page](https://github.com/halostatue/diff-lcs/compare/v1.1.3...v1.5.0).

These are the first 10 commits:
- (7f6f229) [Fix+specs for Diff::LCS::ChangeTypeTests predicates](https://github.com/halostatue/diff-lcs/commit/7f6f229bd94402731552c437874091582e6fe0d7)
- (652cc33) [Merge pull request #8 from ged/master](https://github.com/halostatue/diff-lcs/commit/652cc334c6189fe639c5f54abecdcef2a2833b35)
- (86e4e1b) [Adding autotestability.](https://github.com/halostatue/diff-lcs/commit/86e4e1ba222cab4dddf13caa5c56925c9a6401dc)
- (a341ac7) [Major investigation to Diff::LCS bugs.](https://github.com/halostatue/diff-lcs/commit/a341ac7ca261a73e37c21a7f034f5892ea7dcde4)
- (2151e20) [Starting on a substantial refactoring to fix bugs.](https://github.com/halostatue/diff-lcs/commit/2151e2093dfd00d6aff72f49d13a137efeed2e53)
- (ba57477) [Get rid of useless shebangs.](https://github.com/halostatue/diff-lcs/commit/ba574774574643a59ab3f11b9355b750fd319425)
- (8d6dced) [Merge pull request #9 from voxik/master](https://github.com/halostatue/diff-lcs/commit/8d6dced2e5be35b61ded3ea00f8083a6116a66e0)
- (5e7577b) [Updated GPL2 license file (#10).](https://github.com/halostatue/diff-lcs/commit/5e7577bc40544b3dc715a3e85f0d6075a7dc0c6f)
- (0dc22d2) [Merge pull request #11 from voxik/license](https://github.com/halostatue/diff-lcs/commit/0dc22d241efa69d296b4c19812d51889e243404c)
- (77dba95) [Fixing warnings in Ruby code.](https://github.com/halostatue/diff-lcs/commit/77dba95be2a561e0000d83749a1cdd23ad71b90c)


</details>